### DISCUSSION TITLE
Release schedule

### DISCUSSION THREAD

**Discussion by davewatson91** [hassio-addons/app-tailscale#229] [General]
Hi all! 

Thanks to @frenck for the great work here. I can see that you've got the accept-dns in the main branch, but it hasn't been released. Wondering if there is a known timeframe for when the next release will be? I have a use-case for this, and don't want to jump to another add-on, when this one is well supported :)

Alternatively, is there a way that I can update my add-on to align with the main branch? 

Thanks!

**Comment by lmagyar** [comment]
Until a new version is released, you can use my fork, where all the unreleased changes are available, plus some more unmerged changes. See: https://github.com/lmagyar/homeassistant-addon-tailscale Start only one of these add-ons, better not to mess up the networking under HA.

**Reply by davewatson91** [reply]
Thanks so much - just swapping over now. Do appreciate it!
